src/store/lineup/sagas.js

```diff
@@ -31,11 +32,22 @@ const flatten = list =>
   list.reduce((a, b) => a.concat(Array.isArray(b) ? flatten(b) : b), [])
 function* filterDeletes(tracksMetadata, removeDeleted) {
   const tracks = yield select(getTracks)
-
+  const users = yield select(getUsers)
   return tracksMetadata
     .map(metadata => {
       // If we said to remove deleted tracks and it is deleted, remove it
       if (removeDeleted && metadata.is_delete) return null
+      // If we said to remove deleted and the track/playlist owner is deactivated, remove it
+      else if (
+        removeDeleted &&
+        users[metadata.owner_id]?.is_deactivated === true
```

Comment:
Is the `=== true` necessary? What other truthy value could `is_deactivated` take?